File: Python/client_byDeviceId.py
import ttio_udp 

# by deviceId

productId = 19244
hashStr = '8RLfCQROVqpwag'
deviceId = 'TTIO_TRIAL'
messageStr = 'ABCD1234'
# sha1 = ''

ttio_udp.sendUDPMessageTTIOv0(productId,hashStr,deviceId,messageStr)
# ttio_udp.sendUDPMessageTTIOv0(productId,hash,deviceId,messageStr,sha1)

File: JavaScript/client_byDeviceId.js
const ttio_udp = require('lib/ttio_udp');

// by deviceId

var productId = 1234;
var hash = 'hashexample'
var deviceId = 'TTIO_TRIAL'
var messageStr = 'ABCD1234'
//let sha1 = ''

ttio_udp.sendUDPMessageTTIOv0(productId,hash,deviceId,messageStr)
//ttio_udp.sendUDPMessageTTIOv0(productId,hash,deviceId,messageStr,sha1)
